src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.OpenSsl.cs

```diff
@@ -1077,7 +1078,7 @@ internal static SslException CreateSslException(string message)
             // Capture last error to be consistent with CreateOpenSslCryptographicException
             ulong errorVal = Crypto.ErrPeekLastError();
             Crypto.ErrClearError();
-            string msg = SR.Format(message, Marshal.PtrToStringUTF8(Crypto.ErrReasonErrorString(errorVal)));
+            string msg = SR.Format(message, Utf8StringMarshaller.ConvertToManaged(Crypto.ErrReasonErrorString(errorVal)));
```

Comment:
Change the return type of `ErrReasonErrorString` to `byte*`.

Reply:
There are number of these similar issues that are causing build breaks. Please try to fix all of them by prefering `byte*` as the type for affected fields and local variables.